Sources/RInAppMessaging/DependencyManager/MainContainer.swift

```diff
@@ -10,23 +10,13 @@ internal enum MainContainerFactory {
 
     private typealias ContainerElement = TypedDependencyManager.ContainerElement
 
-    private static func getValidConfigURL() -> URL? {
-        guard !Environment.isUnitTestEnvironment else {
-            return URL(string: "https://config.test")
-        }
-        guard let configURLString = BundleInfo.inAppConfigurationURL, !configURLString.isEmpty else {
-            return nil
-        }
-        return URL(string: configURLString)
-    }
-
-    static func create(dependencyManager manager: TypedDependencyManager) -> TypedDependencyManager.Container {
+    static func create(dependencyManager manager: TypedDependencyManager, configURL: String?) -> TypedDependencyManager.Container {
 
```

Comment:
If configURL is not necessary, can we have: `configURL: String? = nil`?

Reply:
Yeah, I think we can

Reply:
Actually I did some additional refactoring and moved checking validity of config url to `configure()` method. Please take a look at the new commit